### PR TITLE
feat(alerts): Add partnerAlertHitsConnection

### PR DESCRIPTION
**Pull request by damassi**
Addresses https://artsyproduct.atlassian.net/browse/AMBER-914

Adds a new connection to the partners alerts eco system:

```graphql
{
  partner(id: "cypress-test-partner-for-automated-testing-purposes") {
    partnerAlertHitsConnection(first: 10) {
      totalCount
      edges {
        
        internalID
        artwork {
          internalID
          title
        }
        userIDs
        createdAt
        node {
          additionalGeneIDs
        }
      }
    }
  }
}
```

cc @artsy/amber-devs

### CODE REVIEW

File: src/schema/v2/partner/partner.ts [hunk 1]
```diff
@@ -367,6 +386,58 @@ export const PartnerType = new GraphQLObjectType<any, ResolverContext>({
           })
         },
       },
+      partnerAlertsHitsConnection: {
```
Comment (jonallured) [inline src/schema/v2/partner/partner.ts:389]: nit: I would put the plural only on Hit so what about `partnerAlertHitsConnection`?

File: src/schema/v2/partner/partner.ts [hunk 2]
```diff
@@ -367,6 +386,58 @@ export const PartnerType = new GraphQLObjectType<any, ResolverContext>({
...
+          if (!partnerSearchCriteriaHitsLoader) return null
+          const { page, size, offset } = convertConnectionArgsToGravityArgs(
+            args
+          )
+
+          type GravityArgs = {
+            page: number
+            size: number
+            total_count: boolean
+            artist_id?: string
```
Comment (jpotts244) [inline src/schema/v2/partner/partner.ts:413]: I dont think the API is accepting `artist_id` args at the time nor in the way we are persisting in the [schema](https://github.com/artsy/gravity/blob/2049ddd10a5ab8d964e1bec5644b8bc5737d463b/db/schema.rb#L670-L678)

Maybe we burn that bridge when we need to support additional args?

Comment (damassi) [reply src/schema/v2/partner/partner.ts:413]: ahh copy 🍝 ; fixing

File: src/schema/v2/partner/__tests__/partner.test.js [hunk 3]
```diff
@@ -1560,6 +1560,79 @@ describe("Partner type", () => {
...
+      `
+      const partnerSearchCriteriaHitsLoader = () => Promise.resolve(response)
+
+      const data = await runAuthenticatedQuery(query, {
+        ...context,
+        partnerSearchCriteriaHitsLoader,
+      })
+      expect(data).toEqual({
+        partner: {
+          partnerAlertsHitsConnection: {
```
Comment (jpotts244) [inline src/schema/v2/partner/__tests__/partner.test.js:1614]:
```suggestion
          partnerAlertHitsConnection: {
```

**Pull request merged**